style.css

```diff
@@ -93,3 +93,119 @@ main {
   width: 100%;
   background-color: black;
 }
+
+.container {
+  display: grid;
+  place-items: center;
+  padding: 1rem;
+}
+
+h1 {
+  margin-bottom: 25px;
+  line-height: 1.2rem;
+  color: wheat;
+}
+
+.slider {
+  display: block;
+  /* margin: auto; */
+  position: relative;
+  /* display: grid; */
+  /* place-items: center; */
+  max-width: 70%;
+  height: 600px;
+  overflow: hidden;
+}
+
+.slide {
+  display: block;
+  /* display: grid; */
+  /* place-items: center; */
+  height: 100%;
+  user-select: none;
+  display: flex;
+  height: 600px;
+}
+
+.slide img {
+  background: rgb(172, 172, 220);
+  flex: 1 0 100%;
+  object-fit: cover;
+}
+
+.slider__btn-container {
+  z-index: 10000000;
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  /* display: none; */
+}
+
+.slider__btn {
+  border: 10px solid red;
+  border: none;
+  height: 60px;
+  width: 60px;
+  text-align: center;
+  position: absolute;
+  z-index: 1000;
+  top: 48%;
+  border-radius: 50%;
+  display: grid;
+  place-items: center;
+  cursor: pointer;
+  display: none;
+}
+
+.slider__btn--right {
+  opacity: .2;
+  right: 0;
+}
+
+.slider__btn--left {
+  opacity: .2;
+  left: 0;
+}
+
+.slider__btn svg {
+  transform: translateX(-1px);
+  pointer-events: none;
+}
+
+.navigation__bar {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 50px;
+}
+
+.navigation__indicators {
+  margin-left: 20px;
```

Comment:
instead of using margin-left to seperate flex-items use `row-gap or column gap property` in `.navigation__bar`